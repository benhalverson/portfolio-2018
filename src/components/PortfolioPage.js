import React from 'react';
const PortfolioPage = () => (
<div>
  <ul>
    <li>E-Trade Options Analyzer
    <div>I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js,
     a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI</div>
     <div>Add screenshots here</div>
    </li>
    <li>E-Trade Options Income Finder
    <div>I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.</div>
     <div>Add screenshots here</div>
    </li>
    <li>E-Trade Options Backtester
    <div>I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.</div>
     <div>Add screenshots here</div>
    </li>
    <li>BioMarin CLN2 iPad App
    <div>I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique.
     They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out
     the pages and router needed to move between the nav and subnav pages.
    </div>
     <div>Add screenshots here</div>
    </li>
    <li>BioMarin CLN2 iPad App (Unbranded)
    <div>Same app just with a different skin for unbranded marketing purposes.</div>
    </li>
    <li>GSK Vaccines Eduaction Module
    <div>This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline
    given by the client. The technology it used was Backbone.js and SCSS.
    </div>
     <div>Add screenshots here</div>
    </li>
    <li>Jazz Pharmaceuticals</li>
  </ul>
</div>
)

export default PortfolioPage;