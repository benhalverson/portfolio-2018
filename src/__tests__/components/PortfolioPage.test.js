import React from 'react';
import { shallow } from 'enzyme';
import PortfolioPage from '../../components/PortfolioPage';
test('it should render the PortfolioPage', () => {
  const wrapper = shallow(<PortfolioPage/>);
  expect(wrapper).toMatchSnapshot();
});