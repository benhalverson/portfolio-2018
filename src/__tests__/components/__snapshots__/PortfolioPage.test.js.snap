// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the PortfolioPage 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PortfolioPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul>
        <li>
                E-Trade Options Analyzer
                <div>
                        I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI
                </div>
                <div>
                        Add screenshots here
                </div>
        </li>
        <li>
                E-Trade Options Income Finder
                <div>
                        I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
                </div>
                <div>
                        Add screenshots here
                </div>
        </li>
        <li>
                E-Trade Options Backtester
                <div>
                        I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
                </div>
                <div>
                        Add screenshots here
                </div>
        </li>
        <li>
                BioMarin CLN2 iPad App
                <div>
                        I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.
                </div>
                <div>
                        Add screenshots here
                </div>
        </li>
        <li>
                BioMarin CLN2 iPad App (Unbranded)
                <div>
                        Same app just with a different skin for unbranded marketing purposes.
                </div>
        </li>
        <li>
                GSK Vaccines Eduaction Module
                <div>
                        This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
                </div>
                <div>
                        Add screenshots here
                </div>
        </li>
        <li>
                Jazz Pharmaceuticals
        </li>
</ul>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            E-Trade Options Analyzer
            <div>
                        I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI
            </div>
            <div>
                        Add screenshots here
            </div>
</li>,
          <li>
            E-Trade Options Income Finder
            <div>
                        I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
            </div>
            <div>
                        Add screenshots here
            </div>
</li>,
          <li>
            E-Trade Options Backtester
            <div>
                        I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
            </div>
            <div>
                        Add screenshots here
            </div>
</li>,
          <li>
            BioMarin CLN2 iPad App
            <div>
                        I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.
            </div>
            <div>
                        Add screenshots here
            </div>
</li>,
          <li>
            BioMarin CLN2 iPad App (Unbranded)
            <div>
                        Same app just with a different skin for unbranded marketing purposes.
            </div>
</li>,
          <li>
            GSK Vaccines Eduaction Module
            <div>
                        This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
            </div>
            <div>
                        Add screenshots here
            </div>
</li>,
          <li>
            Jazz Pharmaceuticals
</li>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "E-Trade Options Analyzer",
              <div>
                I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI
</div>,
              <div>
                Add screenshots here
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "E-Trade Options Analyzer",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI",
              },
              "ref": null,
              "rendered": "I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Add screenshots here",
              },
              "ref": null,
              "rendered": "Add screenshots here",
              "type": "div",
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "E-Trade Options Income Finder",
              <div>
                I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
</div>,
              <div>
                Add screenshots here
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "E-Trade Options Income Finder",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
              },
              "ref": null,
              "rendered": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Add screenshots here",
              },
              "ref": null,
              "rendered": "Add screenshots here",
              "type": "div",
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "E-Trade Options Backtester",
              <div>
                I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
</div>,
              <div>
                Add screenshots here
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "E-Trade Options Backtester",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
              },
              "ref": null,
              "rendered": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Add screenshots here",
              },
              "ref": null,
              "rendered": "Add screenshots here",
              "type": "div",
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "BioMarin CLN2 iPad App",
              <div>
                I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.
</div>,
              <div>
                Add screenshots here
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "BioMarin CLN2 iPad App",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.",
              },
              "ref": null,
              "rendered": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Add screenshots here",
              },
              "ref": null,
              "rendered": "Add screenshots here",
              "type": "div",
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "BioMarin CLN2 iPad App (Unbranded)",
              <div>
                Same app just with a different skin for unbranded marketing purposes.
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "BioMarin CLN2 iPad App (Unbranded)",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Same app just with a different skin for unbranded marketing purposes.",
              },
              "ref": null,
              "rendered": "Same app just with a different skin for unbranded marketing purposes.",
              "type": "div",
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "GSK Vaccines Eduaction Module",
              <div>
                This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
</div>,
              <div>
                Add screenshots here
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "GSK Vaccines Eduaction Module",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
              },
              "ref": null,
              "rendered": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Add screenshots here",
              },
              "ref": null,
              "rendered": "Add screenshots here",
              "type": "div",
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Jazz Pharmaceuticals",
          },
          "ref": null,
          "rendered": "Jazz Pharmaceuticals",
          "type": "li",
        },
      ],
      "type": "ul",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul>
          <li>
                    E-Trade Options Analyzer
                    <div>
                              I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI
                    </div>
                    <div>
                              Add screenshots here
                    </div>
          </li>
          <li>
                    E-Trade Options Income Finder
                    <div>
                              I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
                    </div>
                    <div>
                              Add screenshots here
                    </div>
          </li>
          <li>
                    E-Trade Options Backtester
                    <div>
                              I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
                    </div>
                    <div>
                              Add screenshots here
                    </div>
          </li>
          <li>
                    BioMarin CLN2 iPad App
                    <div>
                              I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.
                    </div>
                    <div>
                              Add screenshots here
                    </div>
          </li>
          <li>
                    BioMarin CLN2 iPad App (Unbranded)
                    <div>
                              Same app just with a different skin for unbranded marketing purposes.
                    </div>
          </li>
          <li>
                    GSK Vaccines Eduaction Module
                    <div>
                              This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
                    </div>
                    <div>
                              Add screenshots here
                    </div>
          </li>
          <li>
                    Jazz Pharmaceuticals
          </li>
</ul>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              E-Trade Options Analyzer
              <div>
                            I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI
              </div>
              <div>
                            Add screenshots here
              </div>
</li>,
            <li>
              E-Trade Options Income Finder
              <div>
                            I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
              </div>
              <div>
                            Add screenshots here
              </div>
</li>,
            <li>
              E-Trade Options Backtester
              <div>
                            I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
              </div>
              <div>
                            Add screenshots here
              </div>
</li>,
            <li>
              BioMarin CLN2 iPad App
              <div>
                            I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.
              </div>
              <div>
                            Add screenshots here
              </div>
</li>,
            <li>
              BioMarin CLN2 iPad App (Unbranded)
              <div>
                            Same app just with a different skin for unbranded marketing purposes.
              </div>
</li>,
            <li>
              GSK Vaccines Eduaction Module
              <div>
                            This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
              </div>
              <div>
                            Add screenshots here
              </div>
</li>,
            <li>
              Jazz Pharmaceuticals
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "E-Trade Options Analyzer",
                <div>
                  I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI
</div>,
                <div>
                  Add screenshots here
</div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "E-Trade Options Analyzer",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI",
                },
                "ref": null,
                "rendered": "I worked on this project with one other engineer. We rebuilt it from the ground up using Angular.js, a forked version of bootstrap and ui-bootstrap. I primarly worked on the charting part of the UI",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Add screenshots here",
                },
                "ref": null,
                "rendered": "Add screenshots here",
                "type": "div",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "E-Trade Options Income Finder",
                <div>
                  I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
</div>,
                <div>
                  Add screenshots here
</div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "E-Trade Options Income Finder",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
                },
                "ref": null,
                "rendered": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Add screenshots here",
                },
                "ref": null,
                "rendered": "Add screenshots here",
                "type": "div",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "E-Trade Options Backtester",
                <div>
                  I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.
</div>,
                <div>
                  Add screenshots here
</div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "E-Trade Options Backtester",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
                },
                "ref": null,
                "rendered": "I was the lead on this new product. I built the first prototype in Angular.js and version 1 of this product.",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Add screenshots here",
                },
                "ref": null,
                "rendered": "Add screenshots here",
                "type": "div",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "BioMarin CLN2 iPad App",
                <div>
                  I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.
</div>,
                <div>
                  Add screenshots here
</div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "BioMarin CLN2 iPad App",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.",
                },
                "ref": null,
                "rendered": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnav pages.",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Add screenshots here",
                },
                "ref": null,
                "rendered": "Add screenshots here",
                "type": "div",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "BioMarin CLN2 iPad App (Unbranded)",
                <div>
                  Same app just with a different skin for unbranded marketing purposes.
</div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "BioMarin CLN2 iPad App (Unbranded)",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Same app just with a different skin for unbranded marketing purposes.",
                },
                "ref": null,
                "rendered": "Same app just with a different skin for unbranded marketing purposes.",
                "type": "div",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "GSK Vaccines Eduaction Module",
                <div>
                  This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
</div>,
                <div>
                  Add screenshots here
</div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "GSK Vaccines Eduaction Module",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
                },
                "ref": null,
                "rendered": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Add screenshots here",
                },
                "ref": null,
                "rendered": "Add screenshots here",
                "type": "div",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Jazz Pharmaceuticals",
            },
            "ref": null,
            "rendered": "Jazz Pharmaceuticals",
            "type": "li",
          },
        ],
        "type": "ul",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
